eman2/__init__.py

```diff
@@ -142,15 +143,20 @@ def defineBinaries(cls, env):
              SW_EM, '%s/eman-2.31/bin/python' % SW_EM)]
 
         # For Eman3.0.0-alpha
+        eman3_commands = []
+        eman3_commands.append(('wget -c https://github.com/cryoem/eman2/archive/8170d34.tar.gz', "8170d34.tar.gz"))
+        eman3_commands.append(("tar -xvf 8170d34.tar.gz", []))
+        eman3_commands.append(("mv eman2* eman-source", []))
```

Comment:
isn't that gonna move all eman2 folders? Imagine you already have multiple versions installed

Reply:
No Grigory, the folder eman2-xxxxxxxxxxx is a folder that is created inside the eman3-0.0-alpha folder. This does not modify the other installations of the eman binaries

Reply:
The root folder is eman3-0.0-alpha